rules/index.js

```diff
@@ -69,6 +74,19 @@ const plugin = {
       extends: ['plugin:import/warnings'],
       rules: prettierRules,
     },
+    'pretter-ts': {
+      plugins: ['prettier'],
+      extends: ['plugin:import/warnings'],
+      rules: prettierTsRules,
+    },
+    typescript: {
+      plugins: ['@typescript-eslint'],
+      extends: [
+        'plugin:import/warnings',
+        'plugin:@typescript-eslint/recommended',
```

Comment:
Oh just saw you do have this.

Reply:
yeah, i tried to only include ones we wanted to override or increase the severity of